cypress/support/step_definitions/dashboard/users/related-links/i_click_the_link_string_in_the_row_string.js

```diff
@@ -0,0 +1,7 @@
+When("I click the link {string} in the row with header {string}", function (link, header) {
```

Comment:
Looks like there's a mixture of arrow function expressions and function expressions for step callbacks. Would make sense to have consistency but maybe not something for this ticket 🤔

Reply:
mb, need to set up my macros for this device. IntelliJ autofills as function 🙁